storage/client/README.md

```diff
@@ -0,0 +1,365 @@
+- [Azure Subscription][azure_subscription]
+- [Create Strorage Account][storage_account]
+
+### Adding the package to your product
+
+```xml
+<dependency>
+  <groupId>com.azure</groupId>
+  <artifactId>azure-storage</artifactId>
+  <version>12.0.0</version>
+</dependency>
+```
+
+### Create a Storage Account
+To create a Storage Account you can use the Azure Portal or [Azure CLI][azure_cli].
+
+```Powershell
+az group create \
+    --name storage-resource-group \
+    --location westus
+```
+
+### Authenticate the client
+
+In order to interact with the Storage service (Blob, Queue, Message, MessageId, File) you'll need to create an instance of the Service Client class. 
+To make this possible you'll need the Account SAS (shared access signature) string of Storage account. Learn more at [SAS Token][sas_token]
+
+#### Get Credentials
+
+Use the [Azure CLI][azure_cli] snippet below to get the SAS token from the Storage account.
+
+```Powershell
+az storage blob generate-sas
+    --account-name {storage account name}
+    --account-key {storage account key}
+    --container-name {name of blob container}
+    --name {blob name}
+    --permissions {permission to grant}
+    --expiry {date/time to expire SAS token}
+```
+
+```Powershell
+#usage example
+az storage blob generate-sas
+    --account-name azstoragesdkaccount
+    --account-key iseSbogGerMqZj17Xz77XPCPutCqdnDHVOx3ZZsc+UVWfqrrPZRP7bFv64aL8X1Fl06++3JNiuU0iQirr9VCqw==
+    --container-name javablobsettiererror15514606921543
+    --name javablobsettiererror15514606921543
+    --permissions rwdac
+    --expiry 2019-06-05
+```
+Alternatively, get the Account SAS Token from the Azure Portal.
+
+## Key concepts
+### URL format
+Queues are addressable using the following URL format:
+https://<storage account>.queue.core.windows.net/<queue>
+The following URL addresses a queue in the diagram:
+https://myaccount.queue.core.windows.net/images-to-download
+
+#### Resource URI Syntax
+For the storage account, the base URI for queue operations includes the name of the account only:
+```$xslt
+https://myaccount.queue.core.windows.net
+```
+For a queue, the base URI includes the name of the account and the name of the queue:
+```$xslt
+https://myaccount.queue.core.windows.net/myqueue
+```
+
+Each resource may also have components, which are expressed as part of the query string on the base URI. The syntax for the query string is `?comp=<component_name>`. 
+Currently, the following resource components are available for queue resources, as parameters on the URI.
+
+| Component   |      Supported resources      |  Description |
+|:----------:|:-------------:|------|
+| `list` |  Account | Enumerates the entities beneath this resource. On the account, returns the list of queues. |
+| `metadata` |    Queue   | Sets or returns queue properties, including user-defined metadata. |
+
+### Supported HTTP Operations
+Each resource supports operations based on the HTTP verbs GET, PUT, HEAD, and DELETE. 
```

Comment:
Same as the comment for components, this information isn't need to work with the SDK as it will handle it.

Reply:
Removed.

Reply:
I meant remove all mention of supported HTTP operations

Reply:
Done. Good point,